study-builder/studies/pancan/study-workflows.conf

```diff
@@ -69,7 +69,7 @@
         {
           "type": "STUDY_REDIRECT",
           "studyGuid": null,
-          "studyName": "Leiosarcoma Project"
+          "studyName": "Leiomyosarcoma Project"
           "redirectUrl": ${lmsRedirectUrl},
           "expression": ${_pex.is_self}"&&"${_pex.has_lms},
```

Comment:
@ssettipalli Is this a bug? Do we need to redirect to LMS in the child-only case as well?

Reply:
I believe we need to redirect for Leiomyosarcoma child too. ${_pex.is_self} can be removed.

Reply:
Fixed!